modules/core/src/controllers/controller.js

```diff
@@ -263,6 +263,9 @@ export default class Controller {
       return false;
     }
     const pos = this.getCenter(event);
+    if (!this.isPointInBounds(pos) || !this.isDragging()) {
```

Comment:
The `this.isDragging()` check makes sense. However we do want to allow a dragging action to start inside this viewport and continue outside.

Reply:
Make sense. Corrected.